config/crd/bases/ocs.openshift.io_storageconsumers.yaml

```diff
@@ -0,0 +1,78 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why are we starting with v1? lets start with v1alpha1 or v1beta1. This will allow us to reevaluate the API before committing to a specific structure.

Reply:
Proceeded with v1alpha1